tests/test_client.py

```diff
@@ -109,21 +112,65 @@ async def test_trigger_schedule() -> None:
 
 @pytest.mark.asyncio
 async def test_get_schedule() -> None:
+    # todo: relies on https://github.com/pnuckowski/aioresponses/pull/237
     with aioresponses() as m:
-        flexmeasures_client = FlexmeasuresClient("test", "test")
-        flexmeasures_client.access_token = "test-token"
         m.get(
-            "http://localhost:5000/api/v3_0/sensors/3/schedules/schedule-id-string",
+            "http://localhost:5000/api/v3_0/sensors/1/schedules/some-uuid",
+            status=400,
+            payload={"message": "Scheduling job waiting"},
+            repeat=3,
+        )
+        m.get(
+            "http://localhost:5000/api/v3_0/sensors/1/schedules/some-uuid",
             status=200,
-            payload={"test": "test"},
+            payload={
+                "values": [2.15, 3, 2],
+                "start": "2015-06-02T10:00:00+00:00",
+                "duration": "PT45M",
+                "unit": "MW",
+            },
         )
-        await flexmeasures_client.get_schedule(
-            sensor_id=3, schedule_id="schedule-id-string", duration="PT24H"
+        flexmeasures_client = FlexmeasuresClient(
+            "test", "test", request_timeout=2, polling_interval=0.2
         )
-        m.assert_called_once_with(
-            url="http://localhost:5000/api/v3_0/sensors/3/schedules/schedule-id-string",
-            params=None,
-            headers={"Content-Type": "application/json", "Authorization": "test-token"},
-            json={"duration": "P1DT0H0M0S"},
-            ssl=False
+
+        sensor_id = 1
+        schedule_id = "some-uuid"
+        duration = "PT45M"
+
+        schedule, status = await flexmeasures_client.get_schedule(
```

Comment:
Given that they are not too long, here we could pass them as keyword arguments to make it more legible.

Reply:
Changed to set the variables in the function call itself. Mainly we should use a uniform way where possible.